tool/tctl/common/user_command.go

```diff
@@ -139,7 +139,7 @@ func (u *UserCommand) ResetPassword(client auth.ClientI) error {
 func (u *UserCommand) PrintResetPasswordToken(token services.ResetPasswordToken, format string) error {
 	err := u.printResetPasswordToken(token,
 		format,
-		"User %v has been reset. Share this URL with the user to complete password reset, link is valid for %v:\n%v\n\n",
+		"User %q has been reset. Share this URL with the user to complete password reset, link is valid for %v:\n%v\n\n",
```

Comment:
this also emphasizes the user value as otherwise it tends to blend with the rest of the message.